ddtrace/tracer/tracer_test.go

```diff
@@ -667,7 +668,10 @@ func TestTracerAtomicFlush(t *testing.T) {
 	span1.Finish()
 	span2.Finish()
 
-	tracer.forceFlush()
+	// Trigger a flush. Nothing should be flushed, so we can't forceFlush.
+	c := make(chan struct{})
```

Comment:
I would do the same as above, send something down the channel and sleep a bit. The compromise is that we use sleep which is a code smell but we get rid of `c` which is a worse code smell and is in our actual code (not in tests).

Given this, can we please change the type of `flushChan` to `chan struct{}` and remove that `confirm` channel? It will be a very nice clean up.